src/content/tools/mcp-server.md

```diff
@@ -140,18 +145,65 @@ properly by typing `/mcp` in the chat window in Agent mode.
 For more information see the official Gemini Code Assist
 documentation for [using agent mode][].
 
-[instructions]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer#before-you-begin
+[gca-setup]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer#before-you-begin
 [Gemini Code Assist]: https://codeassist.google/
 [Agent mode]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer
 [configure the Gemini CLI]: #gemini-cli
 [using agent mode]: https://developers.google.com/gemini-code-assist/docs/use-agentic-chat-pair-programmer#before-you-begin
 
+### GitHub Copilot in VS Code
+
+:::note
+Support for the Dart MCP server in VS Code required
+v3.116 or later of the [Dart Code extension][].
+:::
+
+By default, the Dart-Code extension uses the
```

Comment:
```suggestion
By default, the Dart extension uses the
```

This looks like an old typo. The extension is Dart, the team who created it is Dart Code.